scipy/linalg/flapack_other.pyf.src

```diff
@@ -2070,11 +2070,11 @@ subroutine <prefix>pftrf(transr,uplo,n,nt,a,info)
     ! where U is an upper triangular matrix and L is lower triangular.
     ! This is the block version of the algorithm, calling Level 3 BLAS.
     !
-    callstatement (*f2py_func)(transr,uplo,&n,a,&info)
+    callstatement (*f2py_func)(&transr,&uplo,&n,a,&info)
     callprotoargument char*,char*,F_INT*,<ctype>*,F_INT*
 
-    character optional,intent(in),check(*transr=='N'||*transr==<'T','T','C','C'>):: transr = 'N'
-    character optional,intent(in),check(*uplo=='U'||*uplo=='L'):: uplo = 'U'
+    character optional,intent(in),check(transr=='N'||transr==<'T','T','C','C'>):: transr = 'N'
+    character optional,intent(in),check(uplo=='U'||uplo=='L'):: uplo = 'U'
```

Comment:
This needs a bit of testing (which is why I opened a draft PR). The check is correct, not sure about the `callstatement` and `callprotoargument`. It might also be `transr` and `char`, and ideally should be `value` in that case to enforce copy instead of referencing.